roofit/roofitcore/test/TestStatistics/testLikelihoodGradientJob.cpp

```diff
@@ -513,5 +512,5 @@ TEST_P(LikelihoodGradientJob, Gaussian1DAlsoWithLikelihoodJob)
    EXPECT_FLOAT_EQ(minNll0, minNll1);
    EXPECT_FLOAT_EQ(mu0, mu1);
    EXPECT_FLOAT_EQ(muerr0, muerr1);
-   EXPECT_NEAR_REL(edm0, edm1, 1e-5);
+   EXPECT_NEAR(edm0, edm1, 1e-5);
```

Comment:
I'm curious about what the motivation was to change from `EXPECT_NEAR_REL` to `EXPECT_NEAR` here. Can you maybe elaborate? Thanks!

Reply:
It's in line with the other edm value test above. I thought the absolute value of the edm is specifically made by Minuit to converge to within 1e-5 in absolute terms, though I admit I'm not 100% sure.

Reply:
Okay, thanks!

Reply:
Perhaps it should actually be 1e-4. That seems to be a default value in Migrad, at least going by this comment: https://github.com/root-project/root/blob/master/math/minuit2/inc/Minuit2/MnApplication.h#L53

Reply:
Hm, now that I'm thinking more about it, I think this is actually not what I was originally trying to test here. As mentioned above, I originally wanted all fits to give *exactly* the same results, also for edm values. When that is no longer necessary, the edm tests don't really add any more value, especially not when all we're testing is a Migrad postcondition that Migrad itself already checks. I think they can be removed in those cases.

Reply:
Or even 1e-3, because RooFit relaxes the tolerance by default:
https://github.com/root-project/root/blob/master/roofit/roofitcore/src/RooMinimizer.cxx#L151

But okay, it's not so important. For me it's okay now that for all tests in this file it's consistent

Reply:
I removed them anyway, and added an explanation in the comment, based on what I said above. I think this makes more sense.